seisbench/models/base.py

```diff
@@ -595,34 +683,67 @@ def annotate(
                                                This is usually acceptable for rotationally invariant models,
                                                e.g., most picking models.
         :type flexible_horizontal_components: bool
+        :param parallelism: If None, uses the `asyncio` implementation. Otherwise, defines the redundancy for each
+                            subjob, i.e., parallelism=2 would start each subjob twice. See the warning above for a
+                            discussion on parallelism for annotate.
+        :type parallelism: None, int
```

Comment:
0 indexing works for this parameter though. So if I enter ```parallelism=0``` I'm still using the multiprocessing approach via ```self._annotate_processes(...)``` and I get back a reasonable input? I can't currently equate this with what's listed in the docstring (starting each subjob 0 times, but getting back something which looks reasonable). 

Just slightly confused with the difference between parallelism=0 and parallelism=1

Reply:
I think `parallelism=0` should be treated as `parallelism=None`. I'll adjust this.